kubernetes/lib/kubernetes/configuration.rb

```diff
@@ -153,6 +153,35 @@ def self.default
       @@default ||= Configuration.new
     end
 
+    def self.default_config()
```

Comment:
Wonder if we should copy the tests from https://github.com/kubernetes-client/javascript/blob/master/src/config_test.ts#L866

Reply:
added.